.eslintrc

```diff
@@ -26,7 +26,11 @@
   "rules": {
     "no-var": "error",
     "prefer-const": "warn",
-    "@typescript-eslint/no-unused-vars": ["warn", { "argsIgnorePattern": "^_" }],
+    "no-misleading-character-class": "warn",
```

Comment:
There was a regex in the rich text code that this didn't like, but didn't seem worth fixing atm.